vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -160,7 +172,8 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 
 	if len(args) == 0 && len(options.Filenames) == 0 {
 		fmt.Fprint(out, "You must specify the type of resource to get. ", valid_resources)
-		return cmdutil.UsageError(cmd, "Required resource not specified.")
+		usageString := fmt.Sprintf("Required resource not specified.\nUse \"%s types\" for a description of each resource.", options.FullCmdName)
```

Comment:
How about proposing `oc explain`? We need to deprecate `oc types` in favor of the former.